src/components/tool/guide/Config/ConfigDuration.js

```diff
@@ -0,0 +1,91 @@
+import React, {useState} from "react";
+import {Alert, AlertIcon, Box, CloseButton, Flex, Heading, Square, Tooltip} from "@chakra-ui/react";
+import {CheckCircleIcon, WarningIcon} from "@chakra-ui/icons";
+import InputSlider from "../../../input/InputSlider";
+
+export default function ConfigDuration({
+                                   title,
+                                   description,
+                                   selected,
+                                   onChange,
+                                   value,
+                                   inputPropsHour,
+                                   inputPropsMinute,
+                                   showAlert = false,
+                                   alertDismissible = true,
+                                   alertType = "info",
+                                   alertMsg = ""
+                               }) {
+    const [alertDismissed, setAlertDismissed] = useState(false);
+    const onClose = () => {
+        setAlertDismissed(true);
+    }
+
+    return (
+        <Box>
+            <Flex direction="column" width="100%">
+                <Flex direction="row" align='center'>
+                    <Box w='120px'>
+                        {selected ?
+                            <CheckCircleIcon color="green.500"/> :
+                            <Tooltip label="Pick an option"><WarningIcon color="orange.500"/></Tooltip>
+                        }
+                        <Tooltip label={description} placement="top" hasArrow>
+                            <Heading size='xs' mb="0" display="inline" ml="2">{title}</Heading>
+                        </Tooltip>
+                    </Box>
+                    <Square size="10px"/>
+                    <Flex style={{width: 'calc(100% - 130px)'}}>
+                        <InputSlider
```

Comment:
I'm not sure sliders are the best UX here, as they don't work so well for large numbers. Can we try with [NumberInputs with steppers](https://v2.chakra-ui.com/docs/components/number-input#usage) instead?

Reply:
We put this on hold to wait for more feedback from other dev if possible.